pyccel/parser/syntactic.py

```diff
@@ -632,6 +632,13 @@ def _visit_FunctionDef(self, stmt):
             for i, arg in enumerate(args):
                 args[i] = str(arg).replace("'", '')
             decorators['stack_array'] = tuple(args)
+
+        if 'allow_negative_index' in decorators:
+            args = list(decorators['allow_negative_index'].args)
+            for i, arg in enumerate(args):
+                args[i] = str(arg).replace("'", '')
```

Comment:
i didn't understand why using `.replace` here

Reply:
You are right. I copied and pasted from the `stack_array` decorator. I suspect it is a hangover from the old parser